jadx-gui/src/main/java/jadx/gui/ui/codearea/SearchBar.java

```diff
@@ -149,8 +167,8 @@ public void toggle() {
 			if (!StringUtils.isEmpty(preferText)) {
 				searchField.setText(preferText);
 			}
-			searchField.requestFocus();
 			searchField.selectAll();
+			searchField.requestFocus();
```

Comment:
This line was changed so that we don't request focus before having all the text selected (for fast keyboard users :P)